eth/eventservices/roundinitializer.go

```diff
@@ -69,26 +98,10 @@ func (r *RoundInitializer) Stop() {
 }
 
 func (r *RoundInitializer) tryInitialize() error {
-	initialized, err := r.client.CurrentRoundInitialized()
```

Comment:
We should keep this check because we don't want to proceed if the round is already initialized (we can't assume that rounds are initialized using the logic in this service).

Reply:
If the current round is initialized this would fail on-chain so wouldn't the gas estimation for this also fail when trying to send the transaction? Saving an RPC call and a check, the current round will be marked "initialized" when receiving a new round event by setting `nextRoundStartBlock`

Reply:
Yeah you're right the gas estimation would fail so we should be able to skip that check. We do end up with a gas estimation error logged for the user in this case and not all ETH clients/providers (AFAIK only geth) support returning the revert reason during gas estimation which might be a bit confusing, but don't feel particularly strongly one way or another about this right now.